apps/pipelines/nodes/nodes.py

```
@@ -275,7 +277,7 @@ def use_session_history(self) -> bool:
     def get_history_mode(self) -> PipelineChatHistoryModes:
         return self.history_mode or PipelineChatHistoryModes.SUMMARIZE
 
-    def get_history(self, session: ExperimentSession) -> list[BaseMessage]:
+    def get_history(self, session: ExperimentSession, exclude_message_id: int = None) -> list[BaseMessage]:
```

Ruff (0.14.10)

PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)